ansible/roles/dev-desktop/files/github_app/readme.md

```diff
@@ -0,0 +1,28 @@
+# How to setup an App
```

Comment:
Thanks for this @oli-obk ! 

Just wanted to clarify something -- will the user (i.e. those that are logging into the cloud compute machine to do Rust development) be setting up the App? Or will an admin of the cloud compute machine set up the App and then admin installs and grants access to the users?

In other words, does a user or admin follow these steps? I feel like it is an admin, but wasn't 100% sure.

Reply:
Yea, this is an admin. I mean, we could make it an app per user, but that's annoying to set up and not really more secure unless you wipe your key after your session and copy it again when you reconnect.

The only step the user has to follow with this is to install the app, for which we'll provide them with a URL, not sure when yet, possibly at login when they haven't set up the app yet